e2e/cypress/e2e/settings/features.cy.ts

```diff
@@ -0,0 +1,216 @@
+        cy.get('cnsl-feature-toggle').should('have.length.greaterThan', 0);
+      });
+    });
+  });
+
+  describe('Feature Modification Tests', () => {
+    afterEach(() => {
+      // Reset features after each feature modification test to ensure clean state
+      apiAuth().then((api) => {
+        resetInstanceFeatures(api);
+      });
+    });
+
+    it('should be able to toggle a feature', () => {
+      // Wait for features to load
+      cy.get('cnsl-feature-toggle').should('be.visible');
+
+      // Get the first feature toggle and check its initial state
+      cy.get('cnsl-feature-toggle')
+        .first()
+        .within(() => {
+          // Find which button is currently checked
+          cy.get('mat-button-toggle.mat-button-toggle-checked').then(($checkedButton) => {
+            // Get all buttons to find the unchecked one
+            cy.get('mat-button-toggle').then(($allButtons) => {
+              // Find the button that is NOT checked
+              const uncheckedButton = $allButtons.not('.mat-button-toggle-checked').first();
+
+              // Click the unchecked button to toggle
+              cy.wrap(uncheckedButton).click();
+
+              // Wait for the save operation
+              cy.wait(1500);
+
+              // Verify the state changed - the previously unchecked button should now be checked
+              cy.wrap(uncheckedButton).should('have.class', 'mat-button-toggle-checked');
+              // The previously checked button should no longer be checked
+              cy.wrap($checkedButton).should('not.have.class', 'mat-button-toggle-checked');
+            });
+          });
+        });
+
+      // Toast messages might not always appear, so make this optional
+      cy.get('body').then(($body) => {
+        if ($body.find('simple-snack-bar, .mat-snack-bar-container').length > 0) {
+          cy.get('simple-snack-bar, .mat-snack-bar-container', { timeout: 5000 }).should('exist');
+        }
+      });
+    });
+
+    it('should handle loginV2 feature toggle', () => {
+      // Check if loginV2 feature toggle exists
+      cy.get('body').then(($body) => {
+        if ($body.find('cnsl-login-v2-feature-toggle').length > 0) {
+          cy.get('cnsl-login-v2-feature-toggle')
+            .should('be.visible')
+            .within(() => {
+              // Should have a feature toggle (with button toggles)
+              cy.get('cnsl-feature-toggle').should('be.visible');
+              cy.get('mat-button-toggle').should('be.visible');
+
+              // Actually toggle the loginV2 feature to test functionality
+              cy.get('mat-button-toggle').first().click();
+
+              // Wait for save operation
+              cy.wait(1500);
+
+              // May have a base URI input field if loginV2 is enabled
+              cy.get('body').then(() => {
+                if (Cypress.$('input[cnslInput]').length > 0) {
+                  cy.get('input[cnslInput]').should('be.visible');
```

Comment:
Does this check if the whole body has SOME cnslInput field that is visible?
Can we check this more precisely, please?
Then, we can also save 1.5 seconds by removing cy.wait